job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/properties/PropertiesContext.groovy

```diff
@@ -102,4 +102,16 @@ class PropertiesContext extends AbstractExtensibleContext {
             delegate.projectUrl(projectUrl ?: '')
         }
     }
+
+    /**
+     * Configures the Build Failure Analyzer.
+     *
+     * @since 1.41
+     */
+    @RequiresPlugin(id = 'build-failure-analyzer', minimumVersion = '1.13')
+    void scannerJobProperty(Boolean doNotScan = false) {
```

Comment:
Probably I misunderstood the way JobDSL convert Jobs XML, and I've coded based on the above xml syntax, that's the reason of adding property to the name of the method. 

``` the configure { project ->
        def properties = project / 'properties'
        properties << 'com.sonyericsson.jenkins.plugins.bfa.model.ScannerJobProperty'(plugin: 'build-failure-analyzer@1.13.0') {
          doNotScan(true)
        }
}
```

I'll review your comments and change that

Cheers